src/app/zap-templates/zcl/data-model/silabs/ha.xml

```diff
@@ -1491,25 +1491,25 @@ limitations under the License.
       <description>
         Indicates lock success or failure
       </description>
-      <arg name="status" type="INT8U"/>
+      <arg name="status" type="Status"/>
```

Comment:
We don't rely on these 'status-code-as-command-response' definitions right? The cluster handler would just call `AddStatusCode` instead of using these right?

Reply:
https://github.com/project-chip/connectedhomeip/issues/10262